cves/CVE-2012-4386.yml

```diff
@@ -144,10 +156,18 @@ interesting_commits:
     by explaining what happened between the VCCs and the fix.
   answer:
   commits:
-    - commit:
-      note:
-    - commit:
-      note:
+    - commit: 32bc45dc42e57f4b488d147d687627017ed9626d
+      note: |
+        This commit claims to "fix" the ExecuteAndWaitInterceptor, but
+        this doesn't fix CVE-2012-4386, it appears that another bug
```

Comment:
Change from "fix CVE-2012-4386, it appears" to "fix CVE-2012-4368. It appears" to make this note read a little more smoothly.

Reply:
Good call here, I'll make that change.